control/server.py

```diff
@@ -30,17 +31,38 @@
 from .state import GatewayState, LocalGatewayState, OmapGatewayState, GatewayStateHandler
 from .grpc import GatewayService
 
-libc = ctypes.CDLL(ctypes.util.find_library("c"))
-PR_SET_PDEATHSIG = 1
+def wait_for_condition(condition_method: Callable[[], bool], error_message=None, timeout=None, interval=1):
```

Comment:
Given you have been so nice to type this signature, why not completely doing it?

```suggestion
def wait_for_condition(condition_method: Callable[[], bool], error_message: Optional[AnyStr]=None, timeout: Optional[float]=None, interval: float=1):
```

Reply:
Thank you Ernesto. Removed [wait_for_condition](https://github.com/ceph/ceph-nvmeof/pull/182#discussion_r1299620078) for now per Mykola comment